### DISCUSSION TITLE
Trouble changing mesh inset

### DISCUSSION THREAD

**Discussion by rickeybobby13** [Jyers/Marlin#2070] [Q&A] [answered]
I just got a sprite pro hot end kit for my ender 3 v2 pro and discovered the right most probe points for the bed level mesh cause the hot end to hit the frame due to the bltouch probe being farther left on the new hot end assembly. From what I've read in the discussions changing the mesh insert should fix this issue but when I try increasing the number nothing seems to change. I changed the name of the printer and the grid from 3x3 to 5x5 and those changes work fine so I don't think I'm compiling or flashing the software wrong. Does the number next to #define MESH_INSET refer to mm? The default is 10 and by my testing an additional 12mm should be enough to fix the issue but even changing the number to 40 didn't seem to effect where the probe was hitting. Is there another setting I need to change?

EDIT: I am using bugfix-JyersUI firmware

**Comment by mriscoc** [comment]


**Reply by rickeybobby13** [reply]
I probably should have clarified in the post but I'm using Bugfix-JyersUI and it looks like the guide you linked if for a different firmware. It did make me think to see if I can mess with the min and max position before compiling to see if that fixes the issue.  It does not appear that I can do this through the settings on the printer like in the guide you linked.

**Reply by mriscoc** [reply]
Sorry I had some tabs opened and I was thinking that I was answering in my Github. Open your configuration.h file and look for your travel limits:
https://github.com/Jyers/Marlin/blob/7b6ff2ce11e614a0c00c472bdfb694fa96d290ab/Marlin/Configuration.h#L1404-L1410

**Comment by rickeybobby13** [comment, marked as answer]
Found the solution! The x axis travel limit had to be adjusted. By default it is set to X_BED_SIZE + 15. Removing the +15 stopped it from hitting the frame.

**Reply by kadi1991** [reply]
Wich firmware version did you use and how do you set up your solution in your firmware? can you maybe send me your firmware version with your upgrade? i got the same problems.... thank you so much!
